target/linux/ath79/dts/qca9563_tplink_tl-wpa8630p-v1.dts

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9563_tplink_tl-wpa8630.dtsi"
+
+/ {
+	compatible = "tplink,tl-wpa8630-v1", "qca,qca9563";
```

Comment:
From what @adschm is saying, this should be `tplink,tl-wpa8630p-v1` (note the `p`, i.e. defining it as a new device in the firmware), and entries should be added in the `boards.d` files (in the same sections as "tl-wpa8630-v1". See previous commits for these wpa8630 dts files to see how those work.

Reply:
Thanks for explaining!